tests/src/system/basic/WskBasicPythonTests.scala

```diff
@@ -14,29 +14,35 @@
  * limitations under the License.
  */
 
-package system.basic;
+package system.basic
 
 import org.junit.runner.RunWith
-import org.scalatest.Matchers
 import org.scalatest.junit.JUnitRunner
-import common.TestUtils
-import common.Wsk
-import common.WskProps
+import org.scalatest.Matchers
 import spray.json._
 import spray.json.DefaultJsonProtocol.StringJsonFormat
+import common.JsHelpers
 import common.TestHelpers
-import common.WskTestHelpers
+import common.TestUtils
+import common.TestUtils.ANY_ERROR_EXIT
+import common.Wsk
 import common.WskProps
+import common.WskTestHelpers
 import common.WhiskProperties
 
 @RunWith(classOf[JUnitRunner])
-class CLIPythonTests
+class WskBasicPythonTests
     extends TestHelpers
     with WskTestHelpers
-    with Matchers {
+    with Matchers
+    with JsHelpers {
 
     implicit val wskprops = WskProps()
     val wsk = new Wsk
+    val zippedPythonAction = Some(TestUtils.getTestActionFilename("action_python_virtualenv.zip"))
```

Comment:
i would prefer if these tests are primarily unit tests (in `PythonActionContainerTests` instead). The CLI tests should then test only that the overall integration works for one example.